docs/dev/overlay-dev-guide.rst

```diff
@@ -263,6 +263,8 @@ the ``Overlays`` tab.
    (``--@ module = true``) and it does not have side effects when loaded as a
    module (i.e. you check ``dfhack_flags.module`` and return before executing
    any statements if the value is ``true``)
+#. you script import `gui` module, even if it is not use it directly:
+    ``local gui = require('gui')``
```

Comment:
are you sure this is true? what situation did you see this with?

Reply:
Without importing `gui` module I was not able to see the `notes` overlay in the control panel overflow list.
Can you try to recreate the issue with empty overlay class?

Reply:
`idle-crafting`, for example, has an overlay widget that does not `require('gui')`

Reply:
`idle-crafting` require `gui.widgets` which itself require `gui`. I will make some test to confirm if this is the case.

Reply:
Can not repeat it, so I closing the PR.
Maybe there was some another factor that I just did not understand earlier.